erpnext/stock/get_item_details.py

```diff
@@ -135,22 +135,44 @@ def get_basic_details(args, item_doc):
 	user_default_warehouse_list = get_user_default_as_list('warehouse')
 	user_default_warehouse = user_default_warehouse_list[0] \
 		if len(user_default_warehouse_list)==1 else ""
+	# Modif CAF
+	argsCustomer =  {
+                doctype: "Customer",
+                name: args.customer
+            }
+	customerData = frappe.client.get(argsCustomer)
+
+	translate_description = ""
+	if item.Description_Table[0].Language == customerData.customer_language:
+		translate_description = item.Description_Table[0].Description
+	if item.Description_Table[1].Language == customerData.customer_language:
+                translate_description = item.Description_Table[1].Description
+	if item.Description_Table[2].Language == customerData.customer_language:
+                translate_description = item.Description_Table[2].Description
+	if item.Description_Table[3].Language == customerData.customer_language:
+                translate_description = item.Description_Table[3].Description
+
```

Comment:
Hi I just modify that part yesterday and now it's working : 
 
# Modif CAF
        customerLanguage = frappe.db.get_value('Customer', args.customer, 'customer_language') # To get the customer language
        translate_description = ""
        base_description = item.get('description_table')
        
#need to replace the 2 if conditions by a for each and a if
if base_description[0].language_multi == customerLanguage:
                translate_description = base_description[0].description_multi
        if base_description[1].language_multi == customerLanguage:
                translate_description = base_description[1].description_multi
        # Fin modif caf

        out = frappe._dict({

                "item_code": item.name,
                "item_name": item.item_name,
                #"description": cstr(item.description_html).strip() or cstr(item.description).strip(),
                "description": cstr(item.description_html).strip() or cstr(translate_description).strip(),


I'll probably try your way before doing a new Update on this.